manifests/local/database-modules.yaml

```diff
@@ -61,6 +63,8 @@ parameters:
         group: networking
         name: basic-networking
         key: PrivateSubnetIds
+  - name: database-name
```

Comment:
repeated block of parameter

Reply:
I define it once for the `rds` module and once for the `postgres` module. Each module needs it defined once.